### ISSUE TITLE
Exampels or tutorial

### ISSUE THREAD

**Issue by Bkeinn** [zakarumych/rapid-qoi#8]
I am very new to rust and would like to use this lib to encode Vec to qoi Images.
I tried to find out how it worked with the benches files, but clear examples like [here](https://github.com/ChevyRay/qoi_rs) would be very nice. (You might wonder why I don't this lib, because it doesn't work/My Image viewer can't open their *.qoi Images).
Thanks.

**Comment by zakarumych** [comment]
Yes, I guess examples would be nice to have. I probably can just copy them from benches with cleanup and adding necessary comments.

**Comment by Bkeinn** [comment]
This would be really helpful.
